config.json

```diff
@@ -66,6 +66,11 @@
             "name": "Automatically send completed eConsents for the forms below to the Remote FHIR Server URL above",
             "type": "checkbox"
         },
+        {
+            "key": "allow-consent-bundle-cleanup",
+            "name": "Allow and clean eConsents messages to the Remote FHIR Server with invalid birth date or name attributes",
```

Comment:
At the moment I'm wondering if a more appropriate `key` might be `allow-econsents-without-identifiers`, and `name` might be `Allow completed eConsents with invalid birth date and/or name attributes`.  I'd be curious to hear your thought process on the current key & name.

Reply:
Frankly I did not spend a lot time into the key naming.
Since the option is set to false as default and we want to preserve the old behaviour as default, clicking the checkbox means that invalid entries (acc. to FHIR specs) of the econsents are dropped (="cleanup" was my wording).
Therefore, I'd suggest:
- `name`: `Allow completed eConsents without birth date and/or name attributes`
- `key`: `allow-econsents-without-identifiers`
Although the key suggests that, if unchecked, identifiers are enforced which is not the case, yet these attributes are simply set to null and are effectively without identifiers as well. Currently, there is no check towards the (semantic) validity of the names / birth date.

Reply:
I think it's worth trying as hard as possible to add settings that make sense long term.  Accounting for existing projects while refactoring poorly designed module settings has been a huge time sink for me in the past.  You make a good point about the lack of enforcement either way.  Would the following be more appropriate?

- `name`: `Remove blank identifiers from eConsents`
- `key`: `remove-blank-econsent-identifiers`

Perhaps you have an even better suggestion?  If that still seems off, we should probably consider tweaking the behavior to match whatever setting name/concept will age more gracefully.

Reply:
Yes, I like that. Let's stick to these phrases.